runbotics/runbotics-orchestrator-ui/src/src-app/components/TreeStructure/CustomTreeItem/CustomTreeItem.tsx

```diff
@@ -0,0 +1,54 @@
+import { forwardRef } from 'react';
+
+import { Box, Tooltip } from '@mui/material';
+
+import { tooltipOptionsWithoutOffset } from '#src-app/components/TooltipIcon/TooltipIcon';
+
+import { StyledCollectionName, StyledTreeItemRoot } from './CustomTreeItem.styles';
+import { CustomTreeItemProps } from './CustomTreeItem.types';
+
+const CustomTreeItem = forwardRef(function StyledTreeItem(
+    props: CustomTreeItemProps,
+    ref: React.Ref<HTMLLIElement>
+) {
+    const { labelIcon: LabelIcon, labelText, ...other } = props;
+
+    return (
+        <StyledTreeItemRoot
+            label={
+                <Box
+                    sx={{
+                        display: 'flex',
+                        justifyContent: 'space-between',
+                        alignItems: 'center',
+                        p: 0.5,
```

Comment:
if pixels then use integers

Reply:
it's not pixels, it's proportion (multiplier) of default mui padding -> so here the 0.5 is 0.5 x 8px (default padding) = 4px